src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/FallbackWindowsRazorProjectHost.cs

```diff
@@ -105,6 +105,17 @@ protected override async Task HandleProjectChangeAsync(string sliceDimensions, I
             return;
         }
 
+        if (TryGetBeforeIntermeidateOutputPath(update.Value.ProjectChanges, out var beforeIntermediateOutputPath))
```

Comment:
I understand the logic about how biop wouldn't be present if it didn't change, but would it make sense to have at least an assert on a compare of the two project keys/iops?